pkgs/by-name/th/thumbor/package.nix

```
@@ -0,0 +1,118 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+  findutils,
+  gifsicle,
+  libjpeg,
+  ffmpeg,
+  nixosTests,
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "thumbor";
+  version = "7.7.7";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "thumbor";
+    repo = "thumbor";
+    tag = version;
+    hash = "sha256-rTQcOkkratFmqXnGkZK/434WmuTXS3Rb+J8jbZ7S6ZA=";
+  };
+
+  build-system = [
+    python3.pkgs.setuptools
+    python3.pkgs.wheel
+  ];
+
+  # Thumbor depends on `remotecv` for queued detection, however `remotecv`
+  # depends on `pyres` which has been removed from nixpkgs because it’s
+  # abandoned. The package still works fine without `remotecv` as long as
+  # queued detection is not configured.
+  dependencies = with python3.pkgs; [
+    colorama
+    derpconf
+    jpegiptc
+    libthumbor
+    piexif
+    pillow
+    pillow-avif-plugin
+    pillow-heif
+    pycurl
+    pytz
+    opencv-python-headless
+    statsd
+    thumbor-plugins-gifv
+    tornado
+    webcolors
+  ];
+
+  optional-dependencies = {
+    svg = [ python3.pkgs.cairosvg ];
+  };
+
+  pythonRelaxDeps = [
+    "pillow"
+    "pytz"
+    "setuptools"
+    "webcolors"
+  ];
+
+  buildInputs = [
+    libjpeg
+    ffmpeg
+  ];
+
+  nativeCheckInputs = [
+    python3.pkgs.pytestCheckHook
+    gifsicle
+    libjpeg
+    ffmpeg
+  ];
+
+  checkInputs = with python3.pkgs; [
+    cairosvg
+    preggy
+    pyssim
+    pytest-asyncio
+    sentry-sdk
+    redis
+  ];
+
+  preCheck = ''
+    # Cleaning up the thumbor directory is necessary to avoid errors with
+    # Python extension modules
+    ${findutils}/bin/find thumbor -type f -name '*.py' -delete
```

```suggestion
    find thumbor -type f -name '*.py' -delete
```
that should already be there and if not go to nativeBuildInputs